crime-analytics.ipynb

```diff
@@ -60,10 +60,8 @@
   },
```

Comment:
<p>@complexsplit check out this cool stuff :)</p>

 _Reply via <a href='https://app.reviewnb.com/cmenguy/crime-analytics/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<p>Looks neat! 😁</p>

Reply:
<p>And check this out too @harleensahni</p>